Feature/UI improvements

pooja-ces opened the pull request.

codecrafter-a committed:
feat : make UI with responsive

codecrafter-a committed:
add components json file

vaishali3012 commented on src/components/ApiStatus.tsx:
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+interface ApiStatusProps {
```
start interface name with capital I

vaishali3012 commented on src/components/Footer.tsx:
```diff
@@ -0,0 +1,30 @@
+                        <li className='text-[#dbd9d9] decoration-0 no-underline'>Status</li>
+                        <li
+                            className='text-[#dbd9d9] decoration-0 no-underline'
+                            onClick={() => router.push("/history")}
```
move common classnames to constants

vaishali3012 commented on src/components/OverallStatus.tsx:
```diff
@@ -0,0 +1,285 @@
+"use client";
+import React, { useState, useEffect } from 'react';
```
one line break

vaishali3012 commented on src/components/OverallStatus.tsx:
```diff
@@ -0,0 +1,285 @@
+import { useData } from '@/api/client';
+import { worldMill } from '@react-jvectormap/world';
+
+
```
remove extra line breaks

vaishali3012 commented on src/components/OverallStatus.tsx:
```diff
@@ -0,0 +1,285 @@
+
+    useEffect(() => {
+        setStatusData(metrics);
+        const defaultData = chartData && chartData.filter((item: { time: string; }) => item.time === 'Today');
```
chartData && chartData.filter ->  chartData?.filter

vaishali3012 commented on src/components/OverallStatus.tsx:
```diff
@@ -0,0 +1,285 @@
+    return (
+        <div className='container px-12 md:px-0 md:mx-auto text-white pt-8'>
+            <Header />
+            <div className='pb-7'>
```
code is too long can we divide into further components

vaishali3012 commented on src/components/SubScribeDialog.tsx:
```diff
@@ -0,0 +1,180 @@
+
+    return (
+        <div>
+            <Dialog>
```
break down code into smaller components

vaishali3012 commented on src/components/ui/button.tsx:
```diff
@@ -0,0 +1,57 @@
+
+import { cn } from "@/lib/utils"
+
+const buttonVariants = cva(
```
can we put into a separate file

pooja-ces committed:
fix=>
=> move common classnames to constants,OverAllStatus file divide into further components
